src/test/kotlin/testhelper/StsMock.kt

```diff
@@ -41,3 +40,9 @@ fun mockAndExpectSTSService(
         .andExpect(MockRestRequestMatchers.header(HttpHeaders.AUTHORIZATION, basicCredentials(username, password)))
         .andRespond(MockRestResponseCreators.withSuccess(json, MediaType.APPLICATION_JSON))
 }
+
+data class StsToken(
+    val access_token: String,
+    val token_type: String,
+    val expires_in: Int
+)
```

Comment:
Kan fjerne mocking av StS-rest også om StsConsumer fjernes her.  Dvs. StsMock.kt og bruk av ting her.

Reply:
Trodde det fortsatt trengtes siden sts fortsatt er konfigurert for et par av soap-kallene, men testene kjører fint uten - så da her jeg ryddet bort STSMock også.